src/conf/startup.properties

```diff
@@ -331,7 +331,8 @@ prism.configstore.listeners=org.apache.falcon.entity.v0.EntityGraph,\
 
 # Setting monitoring plugin, if SMTP parameters is defined
 #*.monitoring.plugins=org.apache.falcon.plugin.DefaultMonitoringPlugin,\
-#                     org.apache.falcon.plugin.EmailNotificationPlugin
+#                     org.apache.falcon.plugin.EmailNotificationPlugin,\
+#                     org.apache.falcon.plugin.FalconDBNotificationPlugin
```

Comment:
Although we haven't been doing it till now, I think adding comments around each plugin about the functionality/ scenario where user will want to enable them, dependencies etc. will improve the usability.